src/index.js

```diff
@@ -84,20 +84,20 @@ const CLASS_NAMES = {
 const ELEMENT_IDS = {
     // CONTACTS_TAB
     addDetailsElementWrapper: "add-details-element-wrapper",
+    addEmail: "email",
```

Comment:
No:
1. What is addEmail? Is it button, input, container?
2. It is field/input
3. I would name it `contactsEmailField`